### PR TITLE
Let's merge loopingz?

### PR DESCRIPTION
**Pull request by Ibmurai**

### CODE REVIEW

File: rancher.py [hunk 1]
```diff
@@ -344,36 +422,119 @@ def log(self, msg, *args, **kwargs):
...
+            environments=dict(required=False, type='list'),
+            github_user=dict(required=False),
+            github_password=dict(required=False),
         ),
         supports_check_mode=True
     ).process()
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (Ibmurai) [inline rancher.py:540]: I like linebreaks at the end of files.

File: rancher.py [hunk 2]
```diff
@@ -344,36 +422,119 @@ def log(self, msg, *args, **kwargs):
...
+            self._facts['api_key'] = {'secret_key': self.params['secret_key'], 'access_key': self.params['access_key']}
             self._account_client = gdapi.Client(url=self.params['url'] + 'v2-beta/',
-                                                    access_key=self.params['access_key'],
-                                                    secret_key=self.params['secret_key'])
-            self._catalog_client = gdapi.Client(url=self.params['url'] + 'v1-catalog/',
                                                 access_key=self.params['access_key'],
                                                 secret_key=self.params['secret_key'])
-            if 'catalogs' in self.params:
+            self._catalog_client = gdapi.Client(url=self.params['url'] + 'v1-catalog/',
+                                                    access_key=self.params['access_key'],
```
Comment (Ibmurai) [inline rancher.py:450]: These two lines are not prettily aligned under the url= parameter.

File: rancher.py [hunk 3]
```diff
@@ -152,6 +166,9 @@ def init_stack(self, stack):
             template = self._catalog_client.by_id_template(url[url.rindex('/') + 1:])
             stack['definition']['rancherCompose'] = template.files['rancher-compose.yml']
             stack['definition']['dockerCompose'] = template.files['docker-compose.yml']
+            
+            if infos[1].startswith('infra*'):
```
Comment (Ibmurai) [inline rancher.py:170]: This could clash unluckily with stupid naming of stuff.

File: rancher.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 from ansible.module_utils.basic import AnsibleModule
 import gdapi
 import json
+import urllib2, base64
 from time import sleep
 
 DOCUMENTATION = '''
```
Comment (Ibmurai) [inline rancher.py:8]: Why add blank DOCUMENTATION?

**Review by Ibmurai — COMMENTED** [review]
Are we sure we want to use python for this?

**Review by henrik-farre — APPROVED** [review]

**Comment by Ibmurai** [comment]
Is this even a thing? Should I just delete this PR?

**Pull request closed without merging**